rfc/pattern_match.qtest

```diff
@@ -0,0 +1,22 @@
+%%command
+php ./src/Main.php %s
```

Comment:
do you wanna create a new repository for `rfc`?

Reply:
I believe a new repository would really be the right way to go.

Reply:
We've had this repo since 2016 😂 
https://github.com/quack/rfc

Reply:
I had noooooo idea of that @joaolucasl! HAHAHAHAHHA

Reply:
Maybe we should do it in there from now on, what do you guys think?

Reply:
I totally approve using `rfc` repository.